src/app/globals.sass

```diff
@@ -0,0 +1,2 @@
+* 
```

Comment:
Do you need this file at alll ?

Reply:
It seems like the easiest way how to get rid of default margin. Should I do it differently?